geolib/models/dsheetpiling/loads.py

```diff
@@ -70,7 +70,7 @@ class UniformLoad(BaseDataClass):
     name: constr(min_length=1, max_length=50)
     left_load: float
     right_load: float
-    verification_load_settings: VerificationLoadSettings = VerificationLoadSettings()
+    verification_load_settings: VerificationLoadSettingsHorizontalLineLoad = VerificationLoadSettingsHorizontalLineLoad()
```

Comment:
The name of VerificationLoadSettingsHorizontalLineLoad now makes no sense anymore, it is applicable for horizontal line loads, uniform loads and surcharge loads. A more generic name, or a class per type would be preferable here

Reply:
I have renamed the VerificationLoadSettings classes to VerificationLoadSettingsLoads (for Surcharge, Uniform and HorizontalLineLoads) and VerificationLoadSettingsMomentNormalForce (for Moment and NormalForce). I have kept the original ones for backwards compatibility. If you want to have those removed, let me know. (or maybe add a deprecation error.

Reply:
Removing the old ones if unused should be fine, we can mention it in the release notes